Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -517,10 +559,22 @@ public override void Update(float frameTime)
             base.Update(frameTime);
 
             var query = EntityQueryEnumerator<ActiveMicrowaveComponent, MicrowaveComponent>();
+
             while (query.MoveNext(out var uid, out var active, out var microwave))
             {
-                //check if there's still cook time left
+
                 active.CookTimeRemaining -= frameTime;
+
+                if (_gameTiming.CurTime >= _targetTime)
+                {
+                    if (!HandleUnsafeItems(uid, microwave))
+                    {
+                        _destruction.BreakEntity(uid);
+                    }
```

Comment:
If you add the `continue` as suggested below, you can move this loop outside of this gameTime if statement.

Reply:
I insist that you adjust this code to match our convention.

Reply:
sure thing